cartography/stats.py

```diff
@@ -53,3 +53,24 @@ def timer(self, stat: str, rate: float = 1.0):
         if self.is_enabled():
             return self._client.timer(stat, rate)
         return None
+
+
+# Global _stats_client
+# Will be set when cartography.config.statsd_enabled is True
+_stats_client: StatsClient = None
+
+
+def set_stats_client(stats_client):
+    """
+    This is used to set the module level stats client configured to talk with a statsd host
+    """
+    global _stats_client
+    _stats_client = stats_client
+
+
+def get_stats(prefix: str) -> ScopedStatsClient:
```

Comment:
Similar to the above comment but probably more important for readability since this will get called as a standalone method. Making it more descriptive for what is being returned would be helpful "get_stats_client" or similar.

Reply:
Updated.